django_testing_utils/mixins.py

```diff
@@ -186,12 +186,12 @@ def update_object(obj: models.Model, *args: Any, **kwargs: Any) -> None:
         """ Update django model object in database only."""
         args_iter = iter(args)
         kwargs.update(dict(zip(args_iter, args_iter)))
-        obj._meta.model.objects.filter(pk=obj.pk).update(**kwargs)
+        obj._meta.model._default_manager.filter(pk=obj.pk).update(**kwargs)
```

Comment:
why?

Reply:
MyPy fails with an error that the model is missing the objects attribute. And it's true.

Reply:
Django model always has "objects" class attribute.

Reply:
May be you need to update mypy-django first

Reply:
> Django model always has "objects" class attribute.

They started removing it in the latest version

https://github.com/typeddjango/django-stubs/blob/028826305cc70bc7943c3bdafdd265f6425f6270/django-stubs/db/models/base.pyi#L37

Perhaps this is related to the fact that "objects" in Django are dynamically added in the metaclass.

Reply:
So, `_meta.model` indeed should be subclass of Model that has `objects` attribute.
Anyway, there is an error in django-stubs or in how type annotations are used here. Using `.objects` is not an error.

Reply:
@tumb1er In Django documentation it is recommended to use "_default_manager" instead of "objects".

https://docs.djangoproject.com/en/4.2/topics/db/managers/#django.db.models.Model._default_manager

> If you’re writing some code that must handle an unknown model, for example, in a third-party app that implements a generic view, use this manager or _base_manager rather than assuming the model has an objects manager.

Django stubs developers also refer to the documentation.
https://github.com/typeddjango/django-stubs/issues/1685#issuecomment-1706829513

Maybe this is still the right solution, switch to "_default_manager"?

Reply:
It might be the right solution, if we have separate model with the only "not_objects" default manager, and a set of tests for it to ensure if all the stuff works correctly.

Reply:
@tumb1er Returned "objects" and disabled this error.